src/classes/Customer.js

```diff
@@ -1,8 +1,35 @@
 class Customer {
   constructor(customer) {
-    this.id = customer.id
+    this.id = customer.id;
     this.name = customer.name;
+    this.allBookings = [];
+    this.previousBookings = [];
+    this.futureBookings = [];
+    this.totalSpent = 0;
+  }
+
+  setCustomerBookings(bookings) {
+    this.allBookings = bookings.filter(booking => booking.userId === this.id)
+    
+    let todaysDate = new Date();
+    todaysDate = todaysDate.toISOString().split('T')[0]
+
+    this.previousBookings = this.allBookings.filter(booking => booking.date < todaysDate)
+
+    this.futureBookings = this.allBookings.filter(booking => booking.date >= todaysDate)
```

Comment:
Looks good, but you will need to format the date to be separated by "/" instead of "-" later on. Might make sense to do it here. (I'll check bookings class too and remove this comment if needed)

Also- do you want to check for current bookings for the day of?